.github/workflows/release.yaml

```diff
@@ -0,0 +1,72 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Desired tag'
+        required: true 
+      tags:
+        description: 'Previous tag'
+        required: true
+jobs:
+  release:
+    if: ${{ github.repository == 'shipwright-io/build' }}
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write  # To be able to get OIDC ID token to sign images.
+      contents: write  # To be able to update releases.
+      packages: write  # To be able to push images and signatures.
+
+    env:
+      IMAGE_HOST: ghcr.io
+      IMAGE_NAMESPACE: ${{ github.repository }}
+      TAG: ${{ github.event.inputs.release }}
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ./.github/actions/setup
+    - uses: sigstore/cosign-installer@v3
+    - uses: azure/setup-helm@v3.5
+      with:
+        version: v3.8.0
+
+    - name: Build Release Changelog
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        PREVIOUS_TAG: ${{ github.event.inputs.tags }}
+      run: |
+        # might not be necessary but make sure
+        chmod +x "${GITHUB_WORKSPACE}/.github/draft_release_notes.sh"
```

Comment:
I agree with this comment. The file should be marked executable in Git and then this goes away.